src/main/resources/META-INF/plugin.xml

```diff
@@ -65,6 +65,7 @@
     <statusBarWidgetFactory id="IdeaVim-Icon" implementation="com.maddyhome.idea.vim.ui.StatusBarIconFactory" order="last, before IdeaVimMode"/>
     <statusBarWidgetFactory id="IdeaVimShowCmd" implementation="com.maddyhome.idea.vim.ui.ShowCmdStatusBarWidgetFactory" order="first"/>
     <statusBarWidgetFactory id="IdeaVimMacro" implementation="com.maddyhome.idea.vim.ui.widgets.macro.MacroWidgetFactory" order="first, after IdeaVimShowCmd"/>
+    <statusBarWidgetFactory id="IdeaVimSearchIndex" implementation="com.maddyhome.idea.vim.extension.searchindex.SearchIndexWidgetFactory" order="after IdeaVimShowCmd" />
```

Comment:
Could you advise if this is an appropriate position for the widget? I'm not entirely sure about the best placement in the status bar hierarchy.

Reply:
We don't have any rules about it now, so I think it's okay to put it there.